test/definitions/checks/disk_speed_minimal_test.rb

```diff
@@ -1,38 +0,0 @@
-require 'test_helper'
```

Comment:
you should be able to use https://github.com/iNecas/foreman_maintain/blob/master/test/definitions/test_helper.rb#L30 for preparing the execution properly, with the dependencies it needs.